runtime/util/search.cpp

```diff
@@ -6,22 +6,16 @@
 #include "runtime/collect.h"
 #include "runtime/header.h"
 
+static std::vector<block *> stepResults;
+
 extern "C" {
 
 void addSearchResult(
     block *result, block ***bufPtr, uint64_t *count, uint64_t *capacity) {
```

Comment:
I dunno if this is because the new_gc branch forgot to do this, or the cherry pick was incomplete, but the type of this function should match the declaration in Decision.cpp